test/data/manifests/fedora-ostree-oci-bootiso.mpp.yaml

```diff
@@ -0,0 +1,428 @@
+version: '2'
```

Comment:
(potentially slightly silly question, sorry!). I don't see a reference to `fedora-ostree-oci-bootiso.json` in the exiting or new code, is this manifest build as part of the tests? If so, what part picks it up (the ostree-images seems to need explicit names in the test case). What am I missing :)

Reply:
There's a makefile target that runs `osbuild-mpp` on all `*.mpp.yaml` files to generate the `.json` manifests.  Then there's a manifest test in gitlab CI that collects them all and builds them.